test/tests/ports/transaction/checkGasPrice.spec.ts

```diff
@@ -70,14 +93,70 @@ test('checkGasPrice component', function ({ components }) {
           })
 
           it('should throw an error', async () => {
-            await expect(checkGasPrice(components)).rejects.toThrow()
+            await expect(
+              checkGasPrice(components, transactionData)
+            ).rejects.toThrow()
           })
         })
       })
 
       describe('and the max gas price allowed is not defined', () => {
         it('should throw an error', async () => {
-          await expect(checkGasPrice(components)).rejects.toThrow()
+          await expect(
+            checkGasPrice(components, transactionData)
+          ).rejects.toThrow()
+        })
+      })
+
+      describe('and the txn contract method is allowed to skip max gas price', () => {
+        describe('and the txn approve the collection manager to spend mana on behalf of the user', () => {
+          it('should not throw an error', async () => {
+            transactionData = {
+              from: '0xe539E0AED3C1971560517D58277f8dd9aC296281',
+              params: [
+                '0x882Da5967c435eA5cC6b09150d55E8304B838f45',
+                '0x0c53c51c000000000000000000000000f7b0e5d753747f102369cc6f8f33cc3feacf7c6200000000000000000000000000000000000000000000000000000000000000a072551c70696250b6de0ec4f3255f3a9e804b06080e2e65e34ac03ea6bddf6df910b461441d1a4ac26f3848e2d65635315891e0d4b4745e02123038616be2a362000000000000000000000000000000000000000000000000000000000000001b0000000000000000000000000000000000000000000000000000000000000044095ea7b3000000000000000000000000e539e0aed3c1971560517d58277f8dd9ac296281ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff00000000000000000000000000000000000000000000000000000000',
+              ],
+            }
```

Comment:
Would you mind moving this into a `beforeEach`? Taking into consideration our testing standards, all objects must be defined in a `beforeEach` to describe the context o the text.